src/project/show.js

```diff
@@ -0,0 +1,41 @@
+window.c.project.Show = (function(m, c, _, models, h){
+  return {
+    controller: function(args) {
+      var vm = m.postgrest.filtersVM({project_id: 'eq'}),
+          idVM = h.idVM,
+          projectDetails = m.prop([]),
+          userDetails = m.prop([]),
+          rewardDetails = m.prop([]);
+
+      vm.project_id(args.root.getAttribute('data-id'));
```

Comment:
Let's add **id** to the project_details so we can always use ```h.idVM``` for the primary key